src/DiffViewerProvider.ts

```diff
@@ -109,4 +96,23 @@ export class DiffViewerProvider implements vscode.CustomReadonlyEditorProvider<D
         .get<boolean>("renderNothingWhenEmpty", false),
     };
   }
+
+  private static updateWebview(args: { diffDocument: vscode.TextDocument; webviewPanel: vscode.WebviewPanel }) {
```

Comment:
Sorry for not making myself clear but I didn't mean to have this method as `static`.

Referring back to my comment, I meant this:
```ts
private updateWebview(args: { diffDocument: vscode.TextDocument; webviewPanel: vscode.WebviewPanel }): void
```
then you can access it with `this.updateWebview` rather than `DiffViewerProvider.updateWebview`.